staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go

```diff
@@ -174,7 +174,9 @@ func (o *CanIOptions) Complete(f cmdutil.Factory, args []string) error {
 				o.ResourceName = resourceTokens[1]
 			}
 		default:
-			return errors.New("you must specify two or three arguments: verb, resource, and optional resourceName")
+			errString := "you must specify two arguments: verb  resource or verb resource/resourceName."
```

Comment:
nit: remove the duplicate space between `verb` and `resource`

Reply:
done